packages/experiments/src/components/Stack/Stack.types.ts

```diff
@@ -61,20 +61,18 @@ export interface IStackProps
    */
   verticalAlign?: 'top' | 'bottom' | Alignment;
 
-  /**
-   * Defines whether the Stack should take up 100% of the width of its parent.
-   */
-  horizontalFill?: boolean;
-
   /**
    * Defines whether the Stack should take up 100% of the height of its parent.
+   * This property is required to be set to true when using the `grow` flag on children elements.
```

Comment:
...for vertical oriented stacks only.

Reply:
By default the Stack is rendered as a block element, which grows to the container already.

Reply:
@khmakoto I merged already but would be good to refine the comment here when you get a moment. At least we'll have updates in on the next publish.